Training5/src/task1/src/actions/todo.js

```diff
@@ -0,0 +1,65 @@
+export const GetTodos = (todos)=>{
+    return {
+        type: 'GET_TODOS',
+        payload: todos,
+    }
+}
+
+export const AddNewTodo = (todo)=>{
+    return {
+        type: 'ADD_NEW_TODO',
+        payload: todo,
+    }
+}
+
+export const SetCompletedTodo = (todo)=>{
+    return {
+        type: 'SET_COMPLETED_TODO',
+        payload: todo,
+    }
+}
+
+export const DeleteTodo = (todo)=>{
```

Comment:
@phuongluongZigvy 
we don't need whole todo in action payload, we only need todoId which one we are going to delete